app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -446,7 +446,7 @@ def center_toolbar_filename_classic
               ems_container ems_middleware container_project container_route container_replicator container_image
               ems_network security_group floating_ip cloud_subnet network_router
               container_image_registry ems_infra flavor host container_build
-              ontap_file_share ontap_logical_disk container_topology middleware_topology
+              ontap_file_share ontap_logical_disk container_topology middleware_topology middleware_server
```

Comment:
not sure how merging/conflicts in a separate branch will be easier from merging/conflicts on master, it seems to me that this is the same challenge. we can and should definitely discuss this.